make-util.js

```diff
@@ -329,8 +329,32 @@ var ensureTool = function (name, versionArgs, validate) {
 }
 exports.ensureTool = ensureTool;
 
+var ensureNvmInstalled = function () {
+    console.log('nvm tool: ');
```

Comment:
>console [](http://example.com/codeflow?start=4&length=7)

[not blocking, can handle in another pr]  we should check for the following: (copy + paste from @ozanaydinMSFT 's comment offline):  

Also; in local environment; developers should update their nvm versions to latest (1.1.11); because "nvm use 20" command is giving error with previous versions